google-cloud-core/src/test/java/com/google/cloud/TimestampTest.java

```diff
@@ -88,6 +88,14 @@ public void ofSqlTimestamp() {
     assertThat(timestamp.toString()).isEqualTo(expectedTimestampString);
   }
 
+  @Test
+  public void ofSqlTimestampOnExactSecond() {
```

Comment:
Can you make this method name more conventional? e.g. testOf_exactSecond or some such.

Reply:
Done. Although this means that the name of this test is not in line with the other test cases in this file, such as for example `ofSqlTimestampPreEpoch`.

Reply:
Section 8.5.2  of the Google style guide addresses this:

When adding new code to a file that is not in Google Style, reformatting the existing code first is recommended, subject to the advice in Section 8.5.1, Reformatting existing code.

If this reformatting is not done, new code is still required to be added in Google Style, even if this creates inconsistencies.

Reply:
I've updated all the test cases for `Timestamp.of(java.sql.Timestamp)` to use consistent naming and `assertEquals` instead of `assertThat...`.